modules/ShipStation/Api/src/Controller/Customer/Api.php

```diff
@@ -250,7 +250,6 @@ protected function getBillingInfo($objOrder = '')
                     $this->addFieldToXML("Country", $objBillingAddress->getCountry()->getCountry());
                     $this->addFieldToXML("CountryCode", $objBillingAddress->getCountry()->getCode());
                 }
-                $this->addFieldToXML("Company", $objBillingAddress->getCompanyName());
```

Comment:
Why was this removed? (Also on L289)

Reply:
im not sure its from the support-5x branch. ill look into it